### ISSUE TITLE
Include a pre-submit job for all-in-one image.

### ISSUE THREAD

**Issue by kishen-v** [ppc64le-cloud/test-infra#478]
A pre-submit job for all-in-one image is needed to ensure that the changes are first validated through a docker build.

This ensures that the images pushed are error free before the post-submit job is triggered.

**Comment by Rajalakshmi-Girish** [comment]
@kishen-v yes can have one instead of manually verifying the docker build success.
